### PR TITLE
Add WebsocketProvider support for Chart Generation Wizard

### PR DESCRIPTION
**Pull request by CharukaK**
## Purpose
Add WebsocketProvider support for Chart Generation Wizard

## Test environment
NPM v5.4.2, Node.JS v8.8.1

### CODE REVIEW

File: components/dashboards-web-component/package-lock.json [hunk 1]
```diff
@@ -2429,7 +2429,7 @@
     "d3": {
       "version": "4.13.0",
       "resolved": "https://registry.npmjs.org/d3/-/d3-4.13.0.tgz",
-      "integrity": "sha512-l8c4+0SldjVKLaE2WG++EQlqD7mh/dmQjvi2L2lKPadAVC+TbJC4ci7Uk9bRi+To0+ansgsS0iWfPjD7DBy+FQ==",
```
Comment (this) [inline components/dashboards-web-component/package-lock.json]: Why `package-lock.json` is changes, without any changes in dependencies

File: components/dashboards-web-component/src/gadgets-generation-wizard/components/ProviderConfigurator.jsx [hunk 2]
```diff
@@ -76,14 +78,27 @@ class ProviderConfigurator extends Component {
...
                             onChange={(id, value) => this.props.handleProviderConfigPropertyChange(id, value)}
                         />
-                        <br/>
+                        <br />
+                    </div>
+                );
+            case (Types.inputFields):
+                return (
+                    <div>
+                        <br />
```
Comment (this) [inline components/dashboards-web-component/src/gadgets-generation-wizard/components/ProviderConfigurator.jsx]: Don't use `<br/>` for padding. Use CSS

File: components/dashboards-web-component/src/gadgets-generation-wizard/components/ProviderConfiguratorComponents/MetaDataCollector.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+ * under the License.
+ */
+import React from 'react';
+import { IconButton, Table, TableBody, TableRow, TableRowColumn } from 'material-ui';
+import ClearButton from 'material-ui/svg-icons/content/clear';
+import AddButton from 'material-ui/svg-icons/content/add';
+import TextInput from '../inputTypes/TextProperty';
+import SelectProperty from '../inputTypes/SelectProperty';
+
+
```
Comment (this) [inline components/dashboards-web-component/src/gadgets-generation-wizard/components/ProviderConfiguratorComponents/MetaDataCollector.jsx]: Unnecessary new line.

File: components/dashboards-web-component/src/gadgets-generation-wizard/components/ProviderConfigurator.jsx (hunk 2 above)
Comment (this) [reply components/dashboards-web-component/src/gadgets-generation-wizard/components/ProviderConfigurator.jsx]: Applies for other places as well.

File: components/dashboards-web-component/package-lock.json (hunk 1 above)
Comment (CharukaK) [reply components/dashboards-web-component/package-lock.json]: This is caused due to rimraf module execution when running a maven clean will fix the issue in another PR

Comment (this) [reply components/dashboards-web-component/package-lock.json]: OK

**Pull request merged**